decide/administration/views.py

```diff
@@ -20,6 +25,115 @@ def index(request):
     return render(request, "build/index.html")
 
 
+class VotingAPI(APIView):
+    permission_classes = (IsAdminAPI,)
+
+    def get(self, request):
+        query = Voting.objects.all()
```

Comment:
why query? maybe _**results**_ or _**votings**_?

Reply:
Query is also fine like it is used in djiango wiki for db operations but for readability has been changed in last commit.